src/include/stir/scatter/ScatterEstimation.h

```diff
@@ -379,6 +381,8 @@ class ScatterEstimation : public ParsingObject
   float min_scale_value;
 
   bool downsample_scanner_bool;
+  int downsampled_detectors_per_ring;
```

Comment:
once we do this, it would make most sense to add the variable for the "dets per ring" as well.

Reply:
oops sorry. yes, num rings